forui/lib/src/foundation/tappable.dart

```diff
@@ -339,6 +336,41 @@ class AnimatedTappableState extends _FTappableState<AnimatedTappable> with Singl
     super.debugFillProperties(properties);
     properties
       ..add(DiagnosticsProperty('controller', controller))
-      ..add(DiagnosticsProperty('animation', animation));
+      ..add(DiagnosticsProperty('animation', animation))
+      ..add(DiagnosticsProperty('style', style));
   }
 }
+
+/// Defines different animation styles for [FTappable] widgets.
+abstract final class FTappableAnimation {
+  // Private constructor to prevent instantiation
+  const FTappableAnimation._();
+
+  /// No animation will be applied.
+  static Tween<double> none = Tween(begin: 1.0, end: 1.0);
+
+  /// A bounce animation that scales the widget down and back up.
+  static Tween<double> bounce = Tween(begin: 1.0, end: 0.97);
+}
```

Comment:
For namespaces I think we should use an extension on never, i.e. `extension FTappableAnimations on Never`.

I also think it should be plural to reflect that.